tutorials/physics/interpolation/physics_interpolation_introduction.rst

```diff
@@ -0,0 +1,131 @@
+
+.. code-block:: python
+
+	x_interpolated = 10 + ((30 - 10) * 0.2)
+	x_interpolated = 10 + 4
+	x_interpolated = 14
+
+Let me break that down a bit.
+
+- We know the x starts from the coordinate on the previous tick (x_prev) which is 10 metres.
+- We know that after the full tick, the difference between the current tick and the previous tick will have been added (x_curr - x_prev) (which is 20 metres).
+- The only thing we need to vary is the proportion of this difference we add, according to how far we are through the physics tick.
+
+.. note:: Although this example interpolates the position, the same thing can be done with the rotation and scale of objects. It is not necessary to know the details as Godot will do all this for you.
+
+Smoothed transformations between physics ticks?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Putting all this together shows that it should be possible to have a nice smooth estimation of the transform of objects between the current and previous physics tick.
+
+But wait, you may have noticed something. If we are interpolating between the current and previous ticks, we are not estimating the position of the object *now*, we are estimating the position of the object in the past. To be exact, we are estimating the position of the object *between 1 and 2 ticks* into the past.
+
+In the past
+^^^^^^^^^^^
+
+What does this mean? This scheme does work, but it does mean we are effectively introducing a delay between what we see on the screen, and where the objects *should* be.
+
+In practice people are not very good at noticing this delay, or rather it is typically not *objectionable*. There are already significant delays involved in games, we just don't typically notice them. The most significant effect is there can be a slight delay to input, which can be a factor in fast twitch games. In some of these fast input situations you may wish to turn off physics interpolation and use a different scheme, or use a high tick rate, which mitagates these delays.
+
+Why look into the past? Why not predict the future?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There is another simple alternative to this scheme, which is instead of interpolating between the previous and current tick, we use maths to *extrapolate* into the future, i.e. try to predict where the object *will be*, rather than show it where it was. This can be done and may be offered as an option in future, but there are some significant downsides.
+
+- The prediction may not be correct, especially when an object collides with another object during the physics tick.
+- Where a prediction was incorrect, the object may extrapolate into an "impossible" position, like inside a wall.
+- Providing the movement speed is slow, these incorrect predictions may not be too much of a problem.
+- When a prediction was incorrect, the object may have to jump or snap back onto the corrected path. This can be visually jarring.
```

Comment:
```suggestion
Why look into the past? Why not predict the future?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

There is an alternative to this scheme, which is: instead of interpolating between the previous and current tick, we use maths to *extrapolate* into the future. We try to predict where the object *will be*, rather than show it where it was. This can be done and may be offered as an option in future, but there are some significant downsides:

- The prediction may not be correct, especially when an object collides with another object during the physics tick.
- Where a prediction was incorrect, the object may extrapolate into an "impossible" position, like inside a wall.
- Providing the movement speed is slow, these incorrect predictions may not be too much of a problem.
- When a prediction was incorrect, the object may have to jump or snap back onto the corrected path. This can be visually jarring.
```